src/esm_runscripts/filedicts.py

```diff
@@ -287,7 +323,7 @@ def __init__(self, full_config: dict, attrs_address: str):
         # possible paths for files:
         location_keys = ["computer", "exp_tree", "run_tree", "work"]
         # initialize the locations and complete paths for all possible locations
-        self.locations = dict.fromkeys(location_keys, None)
+        self.locations = dict.fromkeys(location_keys, pathlib.Path("/dev/null"))
```

Comment:
```suggestion
        self.locations = dict.fromkeys(location_keys, None)
```
That file exists, and if we ever check `if self.locations` it will return True. I think it's dangerous

Reply:
I still don't understand why we initialize `self.locaitons` here at all, to be honest. I think I will just remove it entirely.